config/kubernetes/sig-api-machinery/teams.yaml

```diff
@@ -16,6 +16,22 @@ teams:
     - lavalamp
     - liggitt
     privacy: closed
+  sig-api-machinery-members:
+    description: Members of sig-api-machinery
+    members:
+    - deads2k
+    - derekwaynecarr
```

Comment:
do we have an emeritus category here?  Derek and Dan Mace are probably good to be retired.

Reply:
+1, I think making an api-machinery-emeritus group here would be nice

Reply:
who should go in this group?